metadoc-js/src/main/scala/monaco/Monaco.scala

```diff
@@ -2253,7 +2253,7 @@ package languages {
   sealed trait DocumentHighlightKind extends js.Object {}
 
   @js.native
-  @JSGlobal("monaco.languages.DocumentHighlightKind")
+  @JSGlobal("monaco.languages.TextDocumentHighlightKind")
```

Comment:
Is this to align better with the scalameta API?

Reply:
whoops, that was a sloppy sed rename, good catch! Reverted